docs/descriptor/includes/image/osbs.rst

```diff
@@ -50,6 +49,31 @@ in the :ref:`generation phase<handbook/building/build-process:Generating require
     osbs:
         extra_dir: custom-files
 
+OSBS extra copy command
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Key
+    ``extra_dir_target``
+Required
+    No
+
+If this key is set then the contents of ``extra_dir`` will be copied during the image build to the location specified by this key. If this is *not* set, no ``COPY`` commands will be added. For example, with the value of ``/`` then
+
+.. code-block:: bash
+
+    foobar.yaml
+    manifests
+    metadata
+
+will be copied to the root of the image filesystem, while with the below example a different target directory is chosen.
+
+.. code-block:: yaml
+
+    osbs:
+        extra_dir: custom-files
+        extra_dir_copy: /image-user/tmp
```

Comment:
What is `extra_dir_copy`?

Reply:
That will teach me to write the docs before the code :-) Changing